### PR TITLE
EI-679 Pause and Resume Payments functionality added

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj [hunk 1]
```diff
@@ -31,50 +31,4 @@
     <ProjectReference Include="..\SFA.DAS.EmployerIncentives.Infrastructure\SFA.DAS.EmployerIncentives.Infrastructure.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj:34]: Is it ok to remove these?

Comment (Pauljgraham) [reply src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj:34]: that's a really good question, looking at it again. These mark the cs files as 'to be generated', will correct this thx

Comment (Pauljgraham) [reply src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj:34]: Looking at this again and I really don't think we need to keep these directives in the csproj file. It all appears to work perfectly well without these sections and files are regenerated automatically within the MSBuild process. 

Let me know your thoughts on this Dave

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj:34]: Ok, just wanted to check 👍

**Review by David-B-Read — APPROVED** [review]

**Review by ruslanss — APPROVED** [review]

**Pull request closed without merging**